### PR TITLE
[MIG] website_sale_category_megamenu: Migration to 11.0

### PR DESCRIPTION
**Pull request by remytms**
Hi,

I've ported this module to 11.0. And then I refactored it to use the Customize Menu in the Odoo Website.

Before it looked like this:
before

After:
after

And the Customize menu:
after_customize

Waiting for your comments. :)

### CODE REVIEW

**Review by robinkeunen — APPROVED** [review]
_wrong tab sorry, trying to dismiss this review_

**Comment by simahawk** [comment]
@robinkeunen English please :)

**Comment by sergio-teruel** [comment]
Can we close this PR?

**Comment by remytms** [comment]
@sergio-teruel This PR is not merged yet into OCA, I prefer to leave it open. I'm waiting for reviews. I'm ready to change anything that seems wrong to the community.
Thanks

File: website_sale_category_megamenu/README.rst
Comment (rafaelbn) [inline website_sale_category_megamenu/README.rst:47]: Please change to rafael.blasco@tecnativa.com

File: website_sale_category_megamenu/__manifest__.py
Comment (rafaelbn) [inline website_sale_category_megamenu/__manifest__.py:13]: Hi, please add Tecnativa as we change from Antiun to Tecnativa

**Review by rafaelbn — CHANGES_REQUESTED** [review]
Are you preserving commit  history? Thanks @remytms

**Comment by rafaelbn** [comment]
Maybe @mFlayyan is interested in review this PR and you @remytms could review his PR https://github.com/OCA/e-commerce/pull/127

Remember, per each PR you make you should review 3 PR from others! :smile:

File: website_sale_category_megamenu/README.rst
Comment (rafaelbn) [inline website_sale_category_megamenu/README.rst:46]: This email also doesn't exist any more, just leave the name

**Review by rafaelbn — CHANGES_REQUESTED** [review]
As said there are missing commits from 2015 https://github.com/OCA/e-commerce/pull/82/commits
And from 2016 https://github.com/OCA/e-commerce/pull/135/commits

**Comment by rafaelbn** [comment]
Hi @remytms :smile: 

> @sergio-teruel This PR is not merged yet into OCA, I prefer to leave it open. I'm waiting for reviews. I'm ready to change anything that seems wrong to the community.
Thanks

I reviewed ! thanks you!

**Comment by remytms** [comment]
There is missing commit in my PR. I will not redo all the work. I have started from the 9.0 branch instead of the 8.0 branch. I close this PR.

**Pull request closed without merging**
